hack/golangci-lint.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+PKG=github.com/dragonflyoss/Dragonfly
+LINT_IMAGE=pouchcontainer/pouchlinter:v0.2.3
```

Comment:
In fact, there is no need to use `pouchcontainer/pouchlinter:v0.2.3` which contains a lot of other features and the size of this image is very huge. And you can write a `Dockerfile` and I will build and push it to the repo of [dragonflyoss](https://hub.docker.com/u/dragonflyoss).

Reply:
No, I do not think the `make golangci-lint` command should depend on docker runtime.
If user installed the golangci-lint, it can directly use `make golangci-lint`.

Reply:
Agree. We should try to keep the image light.

Reply:
Yes and I have already opened a PR for this linter image. https://github.com/dragonflyoss/dragonfly-scripts/pull/4